### PR TITLE
Rspamd 1.7.4 worker change

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5478

### CODE REVIEW

File: filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy [hunk 1]
```diff
@@ -17,7 +17,6 @@ worker "rspamd_proxy" \{
 
     # number of worker
     count = {$rspamd{MaxProcesses} || '1'}
-    type = "proxy";
```
Comment (DavidePrincipi) [inline filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: Just out of Curiosity Why this line has not the corresponding Worker section?

Comment (stephdl) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: I really do not know :) 
https://github.com/vstakhov/rspamd/blob/master/conf/worker-proxy.inc

Comment (DavidePrincipi) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: The suggested link still has type=proxy

https://github.com/vstakhov/rspamd/blob/0d217fc8880ed3b6902f72022da7d38f324c368a/conf/worker-proxy.inc#L3

Comment (stephdl) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: yes, in a meanwhile you have this also : [Minor] Remove unnecessary type https://github.com/vstakhov/rspamd/commit/38dfe192fc706ffcad31296105dca7248cca6104

did the main developer missed it :-?

Comment (stephdl) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: it seems also that with 1.7.4 we have more options in the worker-proxy, see https://github.com/vstakhov/rspamd/commit/90f15439af708a6c36a10fa8636f552c9d6b999a

we could add 

- max_retries= 5;
- spam_header = "X-Spam-Flag";

Comment (stephdl) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: question asked for type=proxy on telegram

Comment (stephdl) [reply filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy]: answer of the lead dev , it is a mistake, we could remove `type=proxy`

**Pull request merged**